core/templates/dev/head/pages/exploration_player/AudioBarDirective.js

```diff
@@ -168,6 +169,9 @@ oppia.directive('audioBar', [
 
           $scope.onPlayButtonClicked = function() {
             $scope.hasPressedPlayButtonAtLeastOnce = true;
+            $scope.progressBarIsEnabled =
+                                    $scope.hasPressedPlayButtonAtLeastOnce &&
```

Comment:
Do we need `$scope.hasPressedPlayButtonAtLeastOnce`? I don't think it's used anywhere.

Reply:
Yes sir, i found it there is no use  of it.